recipes/mmligner/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "mmligner" %}
+{% set version = "1.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: http://lcb.infotech.monash.edu.au/{{ name }}/files/{{ name }}_{{ version }}.tar.gz
+  sha256: 0208e8fd1eb0492a13d1bfcba53a7a97f96023f18e3982e334962f0ce1c69f07
+  patches:
+    - add_cli_h_flag.patch
+    - makefile.patch
+
+build:
+  number: 0
+  script: make && mkdir -p ${PREFIX}/bin && cp bin/mmligner ${PREFIX}/bin/mmligner  # [unix]
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - make  # [unix]
+    - jom  # [win]
+    - git  # [win]
```

Comment:
Why do you need ``git`` for windows?

Reply:
It included `sed`, but now we are not providing any Windows packages here (for now) so we can just delete these lines! Thanks for the reminder!